### PR TITLE
chore: add Jenkins documentation to Devops

### LINKED ISSUE #1931: [Add] Want to add Jenkins documentation.
### Site URL

https://www.jenkins.io/doc/

### Description of the above provided site

Jenkins is an open-source automation server that is widely used for building, deploying, and automating software development pipelines. Its documentation is a crucial resource for users, administrators, and developers who want to understand, install, configure, and utilize Jenkins effectively. The most effective and structured documentation I have ever gone is Jenkin's documentation.

### Category

DevOps

### Sub Category

Jenkins

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am participating in Hacktoberfest 2023

### PR DESCRIPTION
**Pull request by goobric**
`Closes #1931`

## Fixes Issue

Closes #1931

## Changes proposed

Added Jenkins documentation to the Devops category

## Note to reviewers

231012_linkhub_jenkins1
231012_linkhub_jenkins2
231012_linkhub_jenkins3

### CODE REVIEW

File: database/devops/jenkins.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[
+    {
+        "name": "Learn how to automate and deploy software by building projects",
+        "description": "The leading open source automation server, Jenkins provides hundreds of plugins to support building, deploying and automating any project.",
```
Comment (CBID2) [inline database/devops/jenkins.json:4]:
```suggestion
"description": "Jenkins provides hundreds of plugins to support building, deploying, and automating any project.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Well done @goobric. Just needs improvement

**Review by Anmol-Baranwal — CHANGES_REQUESTED** [review]
@goobric 
You need to add it here: `database\data.ts`

image


As you can see it's not visible.

image

**Comment by Anmol-Baranwal** [comment]
@goobric 
Why did you delete the head repo?

**Comment by goobric** [comment]
hi @Anmol-Baranwal 
I responded to this message yesterday at about 14:10 UTC, using the github app on my mobile/cell phone; yet my response isn't listed!
I updated the data.ts  file with the 'jenkins' information:
- saved the file locally on my cloned repo
- added the file
- committed the file
- pushed the file
received an error message about 'fast-forward' use 'pull' command from forked repo
tried to 'pull' my forked repo; received another error!
So I thought, "maybe it's better to start again" (clearly this was the wrong approach); so I deleted my locally cloned repo; deleted the forked repo from my github account.
- I forked the original repo to my gh account;
- cloned the repo to my local computer
- created a new branch
- added details about 'jenkins' to index, data.ts, CategoryDescription.ts, added jenkins.json in the devops directory
- saved the files locally
- added the files to the stage
- commited the files 
- pushed the files to my forked repo
Everything pushed successfully to my forked repo
- yet, on my repo; there was no option to "Compare & PR" !
What did I do wrong? please explain to me what I should have done, to have a successful PR?

**Comment by goobric** [comment]
@Anmol-Baranwal what do I do now?
Do I start the process again from the beginning? I am confused!

**Comment by Anmol-Baranwal** [comment]
@goobric Comment on this PR: #1976

**Comment by Anmol-Baranwal** [comment]
I mistakenly created a PR, so the PR will be merged as a co-authored commit.

If you want to only do it urself, then you can create a fresh PR.

For now: it will be similar to this, we both will be assigned to the PR.

image

**Comment by goobric** [comment]
@Anmol-Baranwal that's great :-) 
thanks for co-authoring, I have never co-authored a PR before!
This should be a common option on github (probably is already) I would like to co-author on more PRs; but I don't know how.
I am a big advocate of pair-programming and problem solving.
Thanks for creating the PR and merging it.
Have a wonderful weekend.

**Comment by Anmol-Baranwal** [comment]
It hasn't been merged, you need to comment there in order for me to assign the PR to you.

Then we can go forward.

**Comment by goobric** [comment]
@Anmol-Baranwal 
I am so sorry to ask such a stupid question, but what do I need to say in the comment for the PR to be assigned?
Do I just say, "Please assign this PR me"

231014_LinksHub_PR1
231014_LinksHub_PR2
231014_LinksHub_PR3

**Comment by Anmol-Baranwal** [comment]
@goobric 
Comment anything. GitHub works in a way, that those who have raised or commented can be assigned.

So, it is now showing your option.

As you can see, your name is not available for assignee.

image

**Comment by Anmol-Baranwal** [comment]
See once someone comments.

image

His name is available.
image

**Pull request closed without merging**